### ISSUE TITLE
F841 false positive in async for

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#623]
In GitLab by @behoston on Oct 12, 2017, 06:16

in venv
```bash
pip install flake8
```

```python
async def x(y):
    async for _ in y:
        pass

```
produces: 
>  F841 local variable '_' is assigned to but never used

but not-async works fine 
```python
async def x(y):
    for _ in y:
        pass
```

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 15, 2017, 07:01

As mentioned in our [documentation about filing bugs](http://flake8.pycqa.org/en/latest/internal/contributing.html#filing-a-bug) bug reports for problems with `F*` violations should be reported to github.com/pycqa/pyflakes.

Thanks,
Ian

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 15, 2017, 07:01

closed
